app/views/mappings/index.html.erb

```diff
@@ -1,3 +1,9 @@
+<% if Mapping.has_directory_permission_errors? %>
+  <div class="alert alert-danger" role="alert">
+    <div class="panel-body">Warning: all directories between <code><%= Configuration.home %></code> and <code><%= Configuration.production_database_path %></code> must have their permissions set to <code>0775</code>. Run the following as user <code>bmiadmin</code>:</div>
```

Comment:
I'd remove `Configuration.home` and just say "all directories above #{Configuration.production_database_path}`. Reason is that `Configuration.home` is set to `Pathname.new(ENV['HOME'])` (which is also `Dir.home`) which is incorrect. The database is stored in bmiadmin's home directory, but the PUN process will be each admin user that is in PAS1203 group, not bmiadmin.

So if `efranz` is running the Shiny admin app, `Configuration.home` would be /users/PZS0562/efranz and `Configuration.production_database_path` would be /users/PAS1203/bmiadmin/awesim/share/mappings.sqlite3

That might make ascend preferable, because you don't have to embed `Dir.home('bmiadmin')` in the app.

Or you could do just that: make `Configuration.home` the value of `Dir.home('bmiadmin')`. I think you are setting the ADMIN_USER somewhere (and ADMIN_GROUP?).

Another observation...

```
efranz@owens-login02:/users/PAS1203/bmiadmin/awesim/share$ stat -c "%G" . .. ../.. ../../.. ../../../..
PAS1203
PAS1203
PAS1203
root
root
```

Reply:
That last observation was: could ascend and check permissions if the group is the ADMIN_GROUP. Having the admin group and the admin user known by the app is the "same difference" though... Is there any problem with just checking up the directory tree, regardless of whether the admin user or root owns the directory?